prog/vm/host_nexus.rb

```diff
@@ -74,7 +74,10 @@ def before_run
           total_sockets: st.exitval.fetch("total_sockets"),
           total_dies: st.exitval.fetch("total_dies"),
           total_cores: st.exitval.fetch("total_cores"),
-          total_cpus: st.exitval.fetch("total_cpus")
+          total_cpus: st.exitval.fetch("total_cpus"),
+          # Allocate 2 cores for SPDK. Currently all SPDK installations share
+          # cores 0 and 1.
+          used_cores: 2
```

Comment:
wait, were we simply wrong before by not having used_cores: 1? It's weird that this is de novo?

Reply:
https://github.com/ubicloud/ubicloud/blob/3de9d49b48ed4c20f1fb174e789c80fc5cb178a0/migrate/20230415_add_vm_allocation_counters.rb#L16

Reply:
We reserve one core on each host with this constraint. If we set initial used_cores to 2, we will not use 3 cores in total.

Reply:
huh, seems like we should choose one reservation method...

Reply:
@fdr @enescakir Given the discussions we had here and in other PRs, I just changed the constraint from `used_cores < total_cores` to `used_cores <= total_cores` to unblock 2 cores for SPDK until we implement a better reservation method.

Reply:
sounds reasonable. too bad it didn't work out quite as nicely as it originally seemed.